decidim-proposals/app/views/decidim/proposals/proposals/new.html.erb

```diff
@@ -12,7 +12,7 @@
           </div>
 
           <div class="field hashtags__container">
-            <%= form.text_area :body, rows: 10, class: "js-hashtags", hashtaggable: true %>
+            <%= form.text_area :body, rows: 10, class: "js-hashtags js-mentions", hashtaggable: true %>
```

Comment:
I think that mentions can not be done in the body of a proposal... do you really need to add this css class?

Reply:
removed